mobile-ui/src/main/java/org/buffer/android/boilerplate/ui/model/ArticleViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.buffer.android.boilerplate.ui.model
+
+
+class ArticleViewModel(
```

Comment:
Looks like I'm forgot to tell you about this.

`*ViewModel` seems conflicted with `ViewModel` from Architecture Components

Reply:
How about `*Intent`? @adesamp